third-party/qthread/Makefile

```diff
@@ -158,6 +157,10 @@ ifeq ($(CHPL_LIB_PIC),pic)
 CFLAGS += $(SHARED_LIB_CFLAGS)
 endif
 
+ifneq (, $(CHPL_QTHREAD_DEBUG))
+    CHPL_QTHREAD_CFG_OPTIONS += --enable-debugging --enable-debug=$(CHPL_QTHREAD_DEBUG)
```

Comment:
I'm fine with adding this, but note that you could have achieved this before with `CHPL_QTHREAD_MORE_CFG_OPTIONS="--enable-debugging --enable-debug=<debug options>"` (and most third-parties support this env var to add arbitrary configure options)

Reply:
Removed this.